server/server.go

```diff
@@ -808,6 +1067,9 @@ type Options struct {
 	AckSubsPoolSize    int           // Number of internal subscriptions handling incoming ACKs (0 means one per client's subscription).
 	FTGroupName        string        // Name of the FT Group. A group can be 2 or more servers with a single active server and all sharing the same datastore.
 	Partitioning       bool          // Specify if server only accepts messages/subscriptions on channels defined in StoreLimits.
+	ClusterNodeID      string        // ID of the node within the cluster.
+	ClusterPeers       []string      // Cluster peer IDs.
```

Comment:
I notice there is a `raft.AddPeer(string)` and `raft.RemovePeer(string)` - that may come in useful for dynamic cluster changes someday.